code/API_definitions/device-status.yaml

```diff
@@ -45,6 +45,8 @@ info:
       - `CONNECTED_DATA`, if device is connected to the network via data usage
       - `NOT_CONNECTED`, if device is not connected to the network
 
+    * **LastStatusTime** : This property specifies the time when the status was last checked. Its inclusion in the response indicates that the information may not be current, while its absence suggests that the status information is fresh.
```

Comment:
I think it all depends on the particularities of the use case. The possibility of not knowing the time has not been mentioned so far. I would add information so the developer knows what it means if the property is present or not

Reply:
we can rephrase it as follows: "This property specifies the time when the status was last checked/updated. Its presence in the response indicates the freshness of the information, while its absence implies the information may be outdated or its freshness is uncertain."

Reply:
I would omit the "checked" term, since it is really just an implementation detail if you check and only overwrite the value if is has been changed, or update always. 
Otherwise the text looks good to me.

Reply:
@akoshunyadi @sachinvodafone 
So to be clear, if the information is real-time, according to this, the lastStatusTime should be included?
My understunding was that this field should be present when the information is not fresh but retrieved from another source (a database for example).

Reply:
@fernandopradocabrillo I think if the time is available then it should be included in the response. Missing time info means uncertainty